### PR TITLE
Added Android 5 Support

### PR DESCRIPTION
**Pull request by KillerDogeEmpire**
Blatzar helped me fix the crashing on android 5. I also added new primary colors to QuickNovel

### CODE REVIEW

File: app/src/main/java/com/lagradost/quicknovel/util/InAppUpdater.kt [hunk 1]
```diff
@@ -52,7 +52,7 @@ class InAppUpdater {
 
         private fun Activity.getAppUpdate(): Update {
             try {
-                val url = "https://api.github.com/repos/LagradOst/QuickNovel/releases"
+                val url = "https://api.github.com/repos/KillerDogeEmpire/QuickNovel/releases/latest"
```
Comment (LagradOst) [inline app/src/main/java/com/lagradost/quicknovel/util/InAppUpdater.kt]: This needs to be changed back

**Review by LagradOst — CHANGES_REQUESTED** [review]
This pr looks good 👍 , you only need to change back the update url

File: app/src/main/java/com/lagradost/quicknovel/util/InAppUpdater.kt (hunk 1 above)
Comment (ghost) [reply app/src/main/java/com/lagradost/quicknovel/util/InAppUpdater.kt]: sorry I forgot

**Comment by LagradOst** [comment]
LGTM, ready to merge?

**Comment by KillerDogeEmpire** [comment]
> LGTM, ready to merge?

Yes It's ready

**Pull request merged**